README.md

```diff
@@ -157,6 +157,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [depcon](https://github.com/gondor/depcon) - Depcon is written in Go and allows you to easily deploy Docker containers to Apache Mesos/Marathon, Amazon ECS and Kubernetes.  By [@gonodr][gondor]
 * [deploy](https://github.com/ttiny/deploy) :skull: - Git and Docker deployment tool. A middle ground between simple Docker composition tools and full blown cluster orchestration by [@ttiny](https://github.com/ttiny)
 * [dockit](https://github.com/humblec/dockit) :skull: - Do docker actions and Deploy gluster containers! By [@humblec](https://github.com/humblec)
+* [Grafeas](https://github.com/Grafeas/Grafeas) - Grafeas defines metadata API spec for componentes. By [Grafeas](https://github.com/Grafeas)
```

Comment:
typo in `componentes` ... ?
Would it be possible to have a description that indicates a little bit what it does or what it is used for. It is totally unclear to me what is this project just by reading the description.

Reply:
@veggiemonk This has been updated.